### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by Jammy2211**
Documnetation changes suggested by PyAutoGalaxy astropy review

### CODE REVIEW

File: docs/cookbooks/cookbook_3_multiple_datasets.rst [hunk 1]
```diff
@@ -87,7 +87,7 @@ This means that the model has 7 free parameters in total, the shared ``centre``
     analysis = sum(analysis_list)
 
     analysis = analysis.with_free_parameters(
-        *[model.gaussian.sigma]
+        model.gaussian.sigma]
```
Comment (rhayes777) [inline docs/cookbooks/cookbook_3_multiple_datasets.rst:90]: This is broken syntax

File: docs/cookbooks/cookbook_3_multiple_datasets.rst [hunk 2]
```diff
@@ -139,7 +139,7 @@ We can make multiple parameters free by simply adding them to the input list abo
     analysis = sum(analysis_list)
 
     analysis = analysis.with_free_parameters(
-        *[model.gaussian.sigma, model.gaussian.centre]
+        model.gaussian.sigma, model.gaussian.centre]
```
Comment (rhayes777) [inline docs/cookbooks/cookbook_3_multiple_datasets.rst:142]: Same here

File: docs/overview/multi_datasets.rst [hunk 3]
```diff
@@ -168,7 +168,7 @@ To do that, we interface a model with a summed list of analysis objects
     model = af.Collection(gaussian=af.Model(Gaussian))
 
     analysis = analysis.with_free_parameters(
-        *[model.gaussian.sigma]
+        model.gaussian.sigma]
```
Comment (rhayes777) [inline docs/overview/multi_datasets.rst:171]: and here

File: docs/requirements.txt [hunk 4]
```diff
@@ -1,7 +1,7 @@
 corner==2.2.1
 decorator>=4.2.1
 dill>=0.3.1.1
-dynesty==2.0.1
+dynesty==2.0.2
```
Comment (rhayes777) [inline docs/requirements.txt:4]: Why do we need two identical requirements files?

**Review by rhayes777 — APPROVED** [review]

File: docs/requirements.txt (hunk 4 above)
Comment (Jammy2211) [reply docs/requirements.txt:4]: I could never figure out how to tell readthedocs to look elsewhere,  but it is also true that the docs build has specific requirements (e.g. the template) that the source doesnt.

**Pull request merged**